Nguyen_Ken/Assignments/disappearing_ninjas/server.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, render_template, request, redirect, session, flash
+import re
+
+app = Flask(__name__)
+app.secret_key = 'secretsquirrel'
+
+@app.route('/')
+def index():
+
+    return render_template('index.html')
+
+
+@app.route('/ninja/')
+def ninja():
+
+    return render_template('ninjas.html')
+
+@app.route('/ninja/<vararg>')
```

Comment:
you can call it anything you want. vararg just means variable argument, but you can use any variable name you want.